apps/src/code-studio/pd/application_dashboard/application_dashboard.jsx

```diff
@@ -29,6 +29,7 @@ import AdminCohortView from './admin_cohort_view';
 import _ from 'lodash';
 
 const ROOT_PATH = '/pd/application_dashboard';
+// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
following the tips in the github issue, I think this would be quick to fix with:
```javascript
import {..., useRouterHistory as configureRouterHistory} from 'react-router';
```
I don't think that needs to block this PR, though.

Reply:
Thanks!  I saw this and wasn't sure if it was better or worse to use an alternate name for a method in a common library.  Also interesting to note is that this method is no longer present in more recent versions of react-router.